pkg/assets/create/create_test.go

```diff
@@ -102,17 +102,16 @@ func TestEnsureManifestsCreated(t *testing.T) {
 	}
 
 	// Missing discovery info for kubeapiserverconfig
-	out := &bytes.Buffer{}
 	operatorResource := resourcesForEnsure[0]
 	resourcesForEnsure = resourcesForEnsure[1:]
 	ctx, cancel = context.WithTimeout(context.Background(), 1*time.Second)
 	defer cancel()
-	err = EnsureManifestsCreated(ctx, "testdata", nil, CreateOptions{Verbose: true, StdErr: out})
+	err = EnsureManifestsCreated(ctx, "testdata", nil, CreateOptions{Verbose: true})
 	if err == nil {
 		t.Fatal("expected error creating kubeapiserverconfig resource, got none")
 	}
-	if !strings.Contains(out.String(), "unable to get REST mapping") {
-		t.Fatalf("expected error logged to output when verbose is on, got: %s\n", out.String())
+	if !strings.Contains(err.Error(), "unable to get REST mapping") {
```

Comment:
@mfojtik I should commented earlier: we need a test for this that the error string did not change in apimachinery.

Reply:
@sttts if it change, this test will fail (i don't this we rely on this message in the code).